pkg/broker/filter/stats_reporter.go

```diff
@@ -67,16 +67,18 @@ var (
 	// go.opencensus.io/tag/validate.go. Currently those restrictions are:
 	// - length between 1 and 255 inclusive
 	// - characters are printable US-ASCII
-	triggerFilterTypeKey = tag.MustNewKey(eventingmetrics.LabelFilterType)
-	responseCodeKey      = tag.MustNewKey(eventingmetrics.LabelResponseCode)
-	responseCodeClassKey = tag.MustNewKey(eventingmetrics.LabelResponseCodeClass)
+	triggerFilterTypeKey        = tag.MustNewKey(eventingmetrics.LabelFilterType)
+	triggerFilterRequestTypeKey = tag.MustNewKey("filter_request_type")
```

Comment:
Not sure if we should add this to eventingmetrics package, as this is only relevant in mt-broker case :thinking:

Reply:
Not sure either, here's good for now